apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -38,7 +38,6 @@ const EVENTS = {
   SUBMIT_RP_CONTACT_FORM_EVENT: 'Submit Regional Partner Contact Form',
 
   // Marketing site pages
-  TEACH_PAGE_VISITED_EVENT: 'Teach Page Visited',
```

Comment:
I think we need to keep this event and set it up on the new teach page?

Reply:
I just moved it over to [the file with Amplitude events for .haml files](https://github.com/code-dot-org/code-dot-org/pull/52563/files#diff-545a3f24ef0da9a97406047f870935de5484be31ece2a6e624cc67b33238aa2aR27) since we moved it from a .jsx file (since it was on /courses?view=teacher)!

Reply:
Oh thank you!